### ISSUE TITLE
Using a package/dependency manager

### ISSUE THREAD

**Issue by pyros2097** [matthewmueller/joy#72]
Is just pulled the project and tried to run the tests. But then like most golang projects some dependencies are missing and I needed to manually pull them in like this.
```go
all_test.go:16:2: cannot find package "github.com/sanity-io/litter" in any of:
```
It would be great if we could use a package/dependency manager like dep (which is going to be official) to manage dependencies and version them as it becomes easier for more people to work the this project and not have version clashes.

**Comment by matthewmueller** [comment]
Agreed. I tried dep, but I wasn't able to get it working after about an hour of futzing. Would love some help on that.
